### PR TITLE
Fix NMS box format comment from xyxy to xywh

### LINKED ISSUE #20372: Document description issue for the function non_max_suppression
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Hello, I'm learning the Ultralytics framework. While reading the documentation for the `non_max_suppression` function, I noticed a discrepancy regarding the format description of the `labels` parameter. The documentation states that the input format for labels should be `(class_index, x1, y1, x2, y2)`, but in the actual code, it performs `xywh2xyxy` conversion on it: 

```python
if labels and len(labels[xi]) and not rotated:
            lb = labels[xi]
            v = torch.zeros((len(lb), nc + nm + 4), device=x.device)
            v[:, :4] = xywh2xyxy(lb[:, 1:5])  # box
            v[range(len(lb)), lb[:, 0].long() + 4] = 1.0  # cls
            x = torch.cat((x, v), 0)
```
I'm unsure whether this is a documentation error or if there's another reason for this. Additionally, I've examined the code that calls the `non_max_suppression` function with the `labels` parameter(ultralytics\models\yolo\detect\val.py), and I'm almost certain that the actual format being passed to labels should be `(x,y,w,h)`. This is just a small question that may cause ambiguity.

### Additional

_No response_

### PR DESCRIPTION
**Pull request by nikenake1**
docs: correct labels parameter format from (cls,x1,y1,x2,y2) to (cls,x,y,w,h) #20372 

## 🛠️ PR Summary

<sub>Made with ❤️ by [Ultralytics Actions](https://github.com/ultralytics/actions)<sub>

### 🌟 Summary
Minor documentation updates for improved clarity and accuracy in Ultralytics code and links. 📝✨

### 📊 Key Changes
- Updated a comment link for the Weights & Biases integration to point to the correct documentation page.
- Clarified the format description for label tuples in the `non_max_suppression` function docstring.

### 🎯 Purpose & Impact
- Ensures users and developers are directed to the right integration documentation, reducing confusion.
- Improves code documentation accuracy, making it easier for contributors and users to understand how label data should be structured.
- No changes to core functionality—safe for all users! 🚀

### CODE REVIEW

**Comment by UltralyticsAssistant** [comment]
👋 Hello @nikenake1, thank you for submitting an `ultralytics/ultralytics` 🚀 pull request! This is an automated response to help streamline the review process. An Ultralytics engineer will be with you soon to provide further assistance.

To ensure your documentation update is integrated smoothly, please review the following checklist:

- ✅ **Define a Purpose**: You've already described your update well! If you have any related [issues](https://github.com/ultralytics/ultralytics/issues), consider linking them in the PR description.
- ✅ **Synchronize with Source**: Make sure your PR is up to date with the `ultralytics/ultralytics` `main` branch. If it's behind, update by clicking 'Update branch' or running `git pull` and `git merge main` locally.
- ✅ **Ensure CI Checks Pass**: Please verify all Ultralytics [Continuous Integration (CI)](https://docs.ultralytics.com/help/CI/) checks are passing. Address any issues if they arise.
- ✅ **Update Documentation**: Your change directly improves the documentation—thank you! If your revision impacts other docs, please update those as well in the [documentation](https://docs.ultralytics.com/).
- ✅ **Add Tests**: Documentation changes typically don't require new tests, but if your update affects functionality, please add or update relevant tests and confirm they pass.
- ✅ **Sign the CLA**: If this is your first time contributing, please sign our [Contributor License Agreement (CLA)](https://docs.ultralytics.com/help/CLA/) by commenting "I have read the CLA Document and I sign the CLA".
- ✅ **Minimize Changes**: Confirm that your update contains only the necessary changes for the documentation fix. _"It is not daily increase but daily decrease, hack away the unessential. The closer to the source, the less wastage there is."_ — Bruce Lee

For further details, please refer to our [Contributing Guide](https://docs.ultralytics.com/help/contributing/). If you have any questions, feel free to leave a comment below. Thank you for helping improve the Ultralytics codebase! ✨📝

**Comment by nikenake1** [comment]
I have read the CLA Document and I sign the CLA

**Comment by UltralyticsAssistant** [comment]
🎉 Fantastic work, @nikenake1 and @glenn-jocher! Your attention to detail in refining our documentation makes a genuine difference for everyone in the Ultralytics community. As Leonardo da Vinci said, "Simplicity is the ultimate sophistication." By clarifying links and explanations, you’ve made it even easier for users and contributors to navigate Ultralytics and HUB integrations. Thank you for helping us build a more accessible and user-friendly future with YOLO11 and beyond! 🚀
